### DISCUSSION TITLE
Sometimes can't ping from srsUE to Core

### DISCUSSION THREAD

**Discussion by hoangtuansu** [srsran/srsRAN_Project#225] [General Help]
Hi,

I setup a testbed following the tutorial here:

_https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html_

What I got is a srsUE with IP 10.45.1.2, and the container Open5GS has an interface _ogstun_ with IP 10.45.**0**.1.

This command fails occasasaionly and I have no idea why. It seems there is something going on and I just need to wait for some time before it works again.

_sudo ip netns exec ue1 ping -i 0.1 10.45.1.1_

I tried to troubleshoot and coule not find any host/node/container/... with that IP 10.45.1.1. I also don't get why the container OpenGS has subnet 10.45.0.x instead of 10.45.1.x.

Can anyone elaborate this?
Tuan

**Comment by pgawlowicz** [comment]
Please check the updated tutorial https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html
It contains a description of the routing configuration.
